network-api/networkapi/homepage/factory.py

```diff
@@ -0,0 +1,31 @@
+import factory
+from factory.django import DjangoModelFactory
+
+from networkapi.homepage.models import Homepage, HomepageLeaders, HomepageNews, HomepageHighlights
+from networkapi.highlights.factory import HighlightFactory
+from networkapi.news.factory import NewsFactory
+
+
+class HomepageFactory(DjangoModelFactory):
+    class Meta:
+        model = Homepage
+
+
+class HomepageLeadersFactory(DjangoModelFactory):
+    pass
```

Comment:
why don't we define the fields here?

Reply:
like I said in an earlier comment, this entire branch is stil a WIP. I'll be implementing this today I think

Reply:
ah ok!